R/community.R

```diff
@@ -1041,7 +1041,7 @@ cluster_spinglass <- function(graph, weights = NULL, vertex = NULL, spins = 25,
 #'   weights. Set this to `NA` if the graph was a \sQuote{weight} edge
 #'   attribute, but you don't want to use it for community detection. A larger
 #'   edge weight means a stronger connection for this function.
-#' @param resolution_parameter The resolution parameter to use. Higher
+#' @param resolution The resolution parameter to use. Higher
```

Comment:
you'd need to keep the two docs indeed but for the old one you can add a badge before the meaning https://lifecycle.r-lib.org/reference/badge.html

Example https://github.com/tidyverse/ggplot2/blob/a4be39dee07010c67c31ace059b1f0a24129fc65/R/plot.R#L44 (but here you probably want to use "superseded" not "deprecated" :thinking: )

Reply:
I believe this should now be OK?